python/ray/rllib/optimizers/policy_optimizer.py

```diff
@@ -43,7 +43,7 @@ def make(
         """
 
         local_evaluator = evaluator_cls(*evaluator_args)
-        remote_cls = ray.remote(num_cpus=1)(evaluator_cls)
+        remote_cls = ray.remote(evaluator_cls)
```

Comment:
there should be some way to specify remote args also

Reply:
Added